app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -190,6 +190,22 @@ export const mockExperimentQuery = (
         },
       },
     },
-    data: experiment,
+    data: experiment as getExperiment["experimentBySlug"],
+  };
+};
+
+export const mockNoExperimentQuery = (slug: string) => {
+  return {
+    request: {
+      query: GET_EXPERIMENT_QUERY,
+      variables: {
+        slug,
+      },
+    },
+    result: {
+      data: {
+        experimentBySlug: null,
+      },
+    },
```

Comment:
Huh. I know I wrote this last night, but I wonder if we can just stick all this inside `mockExperimentQuery`? It already has a `modifications` argument, so maybe if we allow you to set it to `null` it skips the Object.assign part and just turns the mock result to `null` a well?

Reply:
If you do agree with this but don't want to re-run CI I can put it in my PR 🤷

Reply:
Oh yeah, that's a good catch. I'll do that real quick here.